openvpn/tunnel/setup_noop.go

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package tunnel
+
+import "github.com/mysteriumnetwork/node/openvpn/config"
+
+// NewNoopSetup creates no-op tunnel setup
+func NewNoopSetup() *noopSetup {
```

Comment:
Maybe just make the struct public, if we are doing nothing in the creation function?

Reply:
👍 I also think we should move to such style. Especially for not over complicated initialisation of structs.